### DISCUSSION TITLE
How can I customise the location in which the logs are sent to?

### DISCUSSION THREAD

**Discussion by gsinhazscaler** [uber-go/zap#1111] [Q&A] [answered]

**Comment by juampynr** [comment, marked as answer]
It's defined by a configuration option: `outputPaths`.

See https://pkg.go.dev/go.uber.org/zap#example-package-BasicConfiguration for an example.
See https://pkg.go.dev/go.uber.org/zap#Config for a short description of `outputPaths`.
